content/docs/guide/gamemodes/climb.md

```diff
@@ -11,16 +11,92 @@ tags:
+Maps made with MomClimb and KZT in mind can have tradtional surf sections. The length and diffculty of these sections is entirely up to the map. Usually they are much shorter (sometimes only 1 ramp) compared to dedicated Surf maps.  
+1.6 Climb has a special version of Surf called **Slide**.  
+1.6 Climb has much lower airacelleration  (10aa compared to MomClimb and KZT's 100aa). This means players can't as easily flick onto a ramp ("board") and genearate speed, or flick off a ramp and generate height.    
+Because of this "Slide" sections are usually brief and focus on keeping on the board rather than gaining enough speed to fly off.
+
+Remember that in any movement mode you can always slide up gentle slopes with just 'W' while looking at the slope!
+
+### Stamina (Jump-Land Penalty) [1.6 Climb Only]
+
+Any time the player jumps (not when they are in the air for any other reason such as falling off something) a timer starts for about 1.31 seconds.
+Whenever the player lands, they will be slowed for however much time is left on the timer.
+For reference, if a player jumps and lands on flat ground about 0.66 seconds or about half the time is taken.
+This means that jumping up to something slows you down longer since you land earlier. And jumping to something beneath will slow you less or not at all.  
+**The slowdown affects both horizontal and vertical movement: running is slower but also your jump height is reduced**.  
+**A Bhop, even a perfect one, does not circumvent the verticle movemnt penalty: Bhops will always get smaller height compared to a normal jump, thus they may not always give you the best distance or be able to reach high places**.  
+For speedrunning getting around this stamnia is a top prority for optimzing runs. For completion players only certain types of platforms make stamina a cause for concern namely ones that are to small or awkward to get a running start from a standsill.
+[This doc goes into futher detail](https://kzguide.gitlab.io/techniques/stamina/).
+
+---
+## Advanced Techniques 
+
+### Climb has some techniques of its own rarely seen outside of its gamemode.
+
+**Fast Ladder Climbing**: Ladders can be climbed by "walking into them". If you are directly facing the ladder hold W, if sideways A/D. If you face the ladder diagonally you can climb the ladder both ways. If you hold W and A/D the speeds add onto each other resulting in a much faster climb.    
+
+**Ladder Jumps**: The speed you gain from fast ladder climbing is enough to fling you into the air! This technique does not have anything to do with inputting jump (which just pushes you off the ladder) and is more simillar to surfing. Gain speed with the climb and flick off near the top. Just like surfing, the way you flick off dictactes your trajectory, and whether you trade speed for height and vice versa.  
```

Comment:
> and is more simillar to surfing

misspelt "similar"

also another important note about ladders: there is no acceleration or deceleration on ladders, so speed changes instantly when you move your mouse or change inputs.